main.lua

```diff
@@ -199,7 +199,7 @@ local function updateRaidList(printErrorOnNoRaid)
             if fullName == currentPlayer.name then 
                 numberColor = '00FF00';
             end
-            raidListText = strtrim(raidListText ..'|cFF' .. numberColor ..tostring(member.karma) ..' - |cFF'.. classColors[member.class:lower()].hex ..member.name ..'\n', "\t\r");
+            raidListText = raidListText ..'|cFF' .. numberColor ..tostring(member.karma) ..' - |cFF'.. classColors[member.class:lower()].hex ..member.name ..'\n|r';

```

Comment:
After like an hour of making sure all the colors, strings, etc. were functioning properly, I had a look at the wiki for ui escape sequences and tried out that "|r". I'll admit I was kinda mad when I found that it was the solution to the text color problem.